### ISSUE TITLE
Carbon site - a11y issues on left menu

### ISSUE THREAD

**Issue by bsonefeld** [carbon-design-system/design-system-website#167]
@arbydev commented 2 hours ago

On http://carbondesignsystem.com the left menu has some keyboard interaction issues.

1. After tabbing past the "Search" field the "Getting Started" text does only a color shift to indicate focus which is not a significant enough change to consider it a highly visible focus indicator. 
2. On your second tab the "What's New" text gets a browser default focus indicator which is not technically accessible since in some browsers it is too light of a blue outline.
3. Third tab moves focus to "Guidelines" which is again only a color change.
4. Fourth tab and 5 tabs on "Style" and "Components" are color changes.
5. After tabbing past "Components" the focus disappears, it's technically moving through the sub-menu items of Components but it should have moved instead to the next node of "Add-Ons"
6. After going through all of "Component's" sub-menu items it shifts to "Add-Ons" with a color change. It does not cycle through "Add-Ons" sub-menu items so that is good.
7. Tabbing past "Add-Ons" results in browser default focus indicators on "Component Status" "Themes" and "Resources.

I would suggest using a treatment like a solid dark border similar to your Tab treatment at the top of each page for a focus indicator across the site. 

> What version of the Carbon Design System are you using? 
8.10.1

## Add labels
a11y

**Comment by marijohannessen** [comment]
@bsonefeld what do you think we should do here? Introduce underline on focus for all menu items?

**Comment by bsonefeld** [comment]
@marijohannessen Yes, I think that's a great suggestion! Let's keep our color change and introduce an underline as well.

**Comment by aagonzales** [comment]
@marijohannessen did we get this added in yet? I agree with adding an underline to both levels in the navigation. 

#### Top level:
![image](https://user-images.githubusercontent.com/11670886/39657168-fd9a12ba-4fca-11e8-8a36-4b8e5bf2655b.png)

#### Secondary level:
![image](https://user-images.githubusercontent.com/11670886/39657192-346c8066-4fcb-11e8-9380-b8c18532e6f9.png)
